gke/standard/zonal/gpu/main.tf

```diff
@@ -0,0 +1,39 @@
+/**
+* Copyright 2024 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+# [START gke_standard_zonal_gpu]
+resource "google_container_cluster" "default" {
+  name               = "gke-standard-regional-gpu"
+  location           = "us-central1-a"
```

Comment:
this means it is zonal, but the name says regional

Reply:
Thanks @yliaog! Yeah, it was a last minute change due to quotas in the CI env and I missed updating the name string.

Reply:
`name` fixed